prog/kube-peers/main.go

```diff
@@ -37,9 +38,9 @@ func getKubePeers() ([]string, error) {
 	for _, peer := range nodeList.Items {
 		var internalIP, externalIP string
 		for _, addr := range peer.Status.Addresses {
-			if addr.Type == "InternalIP" {
+			if addr.Type == "InternalIP" && addressLooksOK(addr.Address) {
```

Comment:
As I say below, there isn't much documentation for what could appear.  Weave will handle a name that resolves to an IPv4 address.  Maybe I'm being over-cautious?